buildpacks/ruby/buildpack.toml

```diff
@@ -1,4 +1,4 @@
-api = "0.8"
+api = "0.9"
```

Comment:
Ah this needs a `CHANGELOG.md` entry, since it affects the tool versions that work with the buildpack (old Pack CLI won't work with the buildpack etc)